tsconfig.esm.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
It would be nice to be able to automate the generation of this file so that the new entry will be added or removed automatically. I'm fine with doing this either in this PR or as a separate PR. Some js script that will find all `tsconfig.esm.json` files in packages folder then will update `references` and create a new file.
WDYT ?

Reply:
Shouldn't the `references` be in the base tsconfig being extended?